### DISCUSSION TITLE
1.6.1-rc.1

### DISCUSSION THREAD

**Discussion by cchafe** [jacktrip/jacktrip#629] [Announcements]
bugfix release (for #628)

## What's Changed
* Updating release manifests for 1.6.0 by @nwang92 in https://github.com/jacktrip/jacktrip/pull/613
* Fixing a few more dark mode icon colors by @nwang92 in https://github.com/jacktrip/jacktrip/pull/614
* Adding more dependencies to linux build instructions on Ubuntu by @nwang92 in https://github.com/jacktrip/jacktrip/pull/617
* Fix plc min hist bug by @cchafe in https://github.com/jacktrip/jacktrip/pull/623
* Set ToS IP header to use DSCP Expedited Forwarding for audio packets by @mikedickey in https://github.com/jacktrip/jacktrip/pull/624
* Fix UDP receive timeout by @psiborg112 in https://github.com/jacktrip/jacktrip/pull/628

**Full Changelog**: https://github.com/jacktrip/jacktrip/compare/v1.6.0...v1.6.1-rc.1

<hr /><em>This discussion was created from the release <a href='https://github.com/jacktrip/jacktrip/releases/tag/v1.6.1-rc.1'>1.6.1-rc.1</a>.</em>

**Comment by cchafe** [comment]
we'd like to release this shortly, any thoughts?

**Comment by cchafe** [comment]
Release steps
- make a pr (dev to main) but checks failed... [Flatpak] (closed #630)
- redo (to include #631 and #632) probably better if it's #633 to stay sequential
- all checks passed
- approved
- in the command line make sure you're on the main branch (git switch main)
- git merge --ff-only origin/dev
- git push
- make a release on github
- ---------------------------------

- copy in the signed assets (looks like this is now automatic?)
- release manifests manual update (Nelson) (1.6.1 is  "stable" rather than edge?)

**Reply by ntonnaett** [reply]
I don't know what went on with the flatpak check, but I've updated the modules and made it (more) coherent with the version in the flathub repo. #631

**Comment by ntonnaett** [comment]
We forgot to update the changelog.yml file. We should put this on the Release steps list.

**Comment by cchafe** [comment]
I woke up remembering that, too this morning! Meanwhile 1.6.1 made it all the way into Arch Linux in about 12 hours!

**Comment by cchafe** [comment]
howTo for the next release cycle:

prelease first, for example, to create 1.6.2-rc.1(v1.6.2-rc.1)
- check that version number in jacktrip_globals.h is updated to 1.6.2-rc.1
- check that changelog.yml is updated for "1.6.2"
- tag & release new (pre)release

actual release, for example,  1.6.2 (v1.6.2)
- check that version number in jacktrip_globals.h is updated to 1.6.2 
- change date in changelog.yml
- make a pr (dev to main) 
- all checks passed
- approved
- in the command line make sure you're on the main branch (git switch main)
- git merge --ff-only origin/dev
- git push
- make a release on github
- signed assets need to be added manually
- release manifests need manual update
